utils/query_helpers.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any
+from langchain.callbacks import get_openai_callback
+from yolopandas import pd
+
+
+def run_query_with_cost(df: pd.DataFrame, query: str, yolo: bool = False):
+    """
+    A function to run a YOLOPandas query with cost estimation returned for your query in terms of tokens used. This includes total tokens, prompt tokens, completion tokens, and the total cost in USD.
```

Comment:
Would you mind adding line breaks in the docstrings to keep lines under 100 characters? `black` does not auto-format this currently.

Reply:
Yep addressed in the latest commit.